### DISCUSSION TITLE
Need help with using DBeaver PostgreSQL

### DISCUSSION THREAD

**Discussion by nilikadam39** [dbeaver/dbeaver#10794] [Q&A] [answered]
For some reason (using the latest version of 7.2.0 DBeaver) I can't for the life of me find the option to create a new table/view. I've googled around and found guides but wherever the guide shows right clicking giving you the "create new table" option, I don't have it on my end.

New to all of this stuff so getting stuck on such a basic/simple first step is beyond frustrating. Someone help please!

**Comment by p91paul** [comment]
A screen recording would be useful to see where you are clicking. It should work by right clicking on the tables/views folders.

**Comment by uslss** [comment, marked as answer]
Hi @nilikadam39 
Probably, you are using a Simple connection view. Try to change it to Advanced in Connection context menu.
![image](https://user-images.githubusercontent.com/31996417/103273808-3cae9900-49d1-11eb-99fc-1169afab3dba.png)
